### DISCUSSION TITLE
Make logger behave more in line with console.log

### DISCUSSION THREAD

**Discussion by CodingDive** [adonisjs/core#4333] [Ideas]
From the [docs](https://v6-alpha.adonisjs.com/docs/logger)
`logger.info('Fetching user by id %s', params.id)`

However, coming from a lot of frontend development, I often see myself using it like this
`logger.info('Fetching user by id', params.id)`

This usually leads to the params.id being simply ignored.

**Comment by RomainLanz** [comment]
Hey @CodingDive! 👋🏻 

Thank you for your input. The logger syntax we use, although based on Pino, is actually a standard approach in the wider logging community, not exclusive to Pino. We understand the appeal of a syntax similar to console.log. However, our current syntax is chosen for its broader compatibility and familiarity among developers.
